src/algorithm.jl

```diff
@@ -112,11 +112,33 @@ algorithm `algo` with verbose level `verbose`.
 """
 function sample_scenarios end
 
+function sample_scenarios(sp::AbstractStochasticProgram, n::Int, depth_max = 1000,to::TimerOutput=TimerOutput(), verbose::Int=0)
+    scenarios = []
+    for i = 1:n
+        push!(scenarios,sample_scenario(sp, depth_max))
+    end
+    return scenarios
+end
+
+function sample_scenario(sp::AbstractStochasticProgram, depth_max = 1000,to::TimerOutput=TimerOutput(), verbose::Int=0)
+    node = get(sp,MasterNode())
+    s = []
```

Comment:
I would say that `s` is not that explicit :p 
Is it possible to type the array?